docker-compose.override.yaml

```diff
@@ -1,86 +1,58 @@
+      "django-admin",
+      "runserver", "0.0.0.0:8000"
+    ]
+    # Log printing via Rich is enhanced by a TTY
+    tty: true
+    env_file: ./dev/.env.docker-compose
     environment:
-      RDWATCH_MINIO_STORAGE_ACCESS_KEY: minio
-      RDWATCH_MINIO_STORAGE_SECRET_KEY: ${RDWATCH_SECRET_KEY:?}
-      RDWATCH_MINIO_STORAGE_ENDPOINT: minio:9002
-      RDWATCH_STORAGE_BUCKET_NAME: rdwatch
-    depends_on:
-      - postgresql
-      - redis-master
-      - minio
-    volumes: # be sure to run docker compose run --rm poetry install --all-extras first
-      - .:/app # for local DEV and minio-storage dev dependency
-
-  # PostgreSQL 14
-  postgresql:
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID:?}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY:?}
+      RDWATCH_SMART_STAC_URL: ${RDWATCH_SMART_STAC_URL:?}
+      RDWATCH_SMART_STAC_KEY: ${RDWATCH_SMART_STAC_KEY:?}
+    volumes:
+      - ./django:/app/django
     ports:
-      - 5432:5432 # expose ports locally
-
-  celery: # jobs for download images
-    environment:
-      RDWATCH_MINIO_STORAGE_ACCESS_KEY: minio
-      RDWATCH_MINIO_STORAGE_SECRET_KEY: ${RDWATCH_SECRET_KEY:?}
-      RDWATCH_MINIO_STORAGE_ENDPOINT: minio:9002
-      RDWATCH_STORAGE_BUCKET_NAME: rdwatch
+      - 8000:8000
     depends_on:
       - postgresql
+      - rabbitmq
       - redis-master
       - minio
 
-  poetry:
-    environment:
-      RDWATCH_MINIO_STORAGE_ACCESS_KEY: minio
-      RDWATCH_MINIO_STORAGE_SECRET_KEY: ${RDWATCH_SECRET_KEY:?}
-      RDWATCH_MINIO_STORAGE_ENDPOINT: minio:9002
-      RDWATCH_STORAGE_BUCKET_NAME: rdwatch
-
-
-  minio:
-    image: minio/minio:latest
-    # When run with a TTY, minio prints credentials on startup
+  celery:
+    build:
+      dockerfile: Dockerfile
+      target: builder
+    command: [
+      "poetry", "run",
+      "--directory", "/app/django",
+      "celery",
+      "--app", "rdwatch.celery",
+      "worker",
+      "--loglevel", "INFO",
+      "--without-heartbeat",
+    ]
+    # Log printing via Rich is enhanced by a TTY
     tty: true
-    command: ["server", "/data", "--console-address", ":${DOCKER_MINIO_CONSOLE_PORT-9003}", "--address", ":${DOCKER_MINIO_PORT-9002}"]
-    volumes:
-      - minio:/data
-    environment:
-      MINIO_ROOT_USER: minio
-      MINIO_ROOT_PASSWORD: ${RDWATCH_SECRET_KEY:?}
-    ports:
-      - ${DOCKER_MINIO_PORT-9002}:9002
-      - ${DOCKER_MINIO_CONSOLE_PORT-9003}:9003
-
-  django:
+    env_file: ./dev/.env.docker-compose
     environment:
-      RDWATCH_MINIO_STORAGE_ACCESS_KEY: minio
-      RDWATCH_MINIO_STORAGE_SECRET_KEY: ${RDWATCH_SECRET_KEY:?}
-      RDWATCH_MINIO_STORAGE_ENDPOINT: minio:9002
-      RDWATCH_STORAGE_BUCKET_NAME: rdwatch
-    profiles:
-      - debug
-      - tools
-      - vscode
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID:?}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY:?}
+      RDWATCH_SMART_STAC_URL: ${RDWATCH_SMART_STAC_URL:?}
+      RDWATCH_SMART_STAC_KEY: ${RDWATCH_SMART_STAC_KEY:?}
```

Comment:
I think we need the broker URL in here
`RDWATCH_CELERY_BROKER_URL: ${RDWATCH_CELERY_BROKER_URL:?}`

Reply:
This is added in the `.env.docker-compose` and `.env.docker-compose-native` files.